actors/builtin/reward/reward_state.go

```diff
@@ -14,14 +14,30 @@ type NetworkTime = big.Int
 type Spacetime = big.Int
 
 type State struct {
-	BaselinePower        abi.StoragePower
-	RealizedPower        abi.StoragePower
-	CumsumBaseline       Spacetime
-	CumsumRealized       Spacetime
+	// Baseline network power in raw bytes (fully deterministic & functionally equivalent to a hard-coded
+	// lookup table), above which, full reward minting will be possible.
+	BaselinePower abi.StoragePower
+
+	// Realized network power in raw bytes.
+	RealizedPower abi.StoragePower
+
+	// Cumulative baseline network power in raw bytes.
+	CumsumBaseline Spacetime
+
+	// Cumulative capped realized power, where capped realized power is min(realized power, baseline power).
+	CumsumRealized Spacetime
+
+	// Captures the notion of how much the network has progressed in its baseline and in advancing network time.
+	// Defined as the unique “time” theta such that CumsumRealized(t) = CumsumBaseline(theta).
 	EffectiveNetworkTime NetworkTime
 
-	SimpleSupply   abi.TokenAmount // current supply
-	BaselineSupply abi.TokenAmount // current supply
+	// FIL supply from SimpleExponentialMinting
```

Comment:
Explain "supply" as allocation/target, as contrasted with amount actually minted. Clarify whether it's the this-epoch expectation, or the integral of that over time.